### PR TITLE
Information modals for notification preferences

### PR DESCRIPTION
**Pull request by LomyW**
Informations for users about notification settings.

### CODE REVIEW

File: src/components/notifications/preferences/PreferenceItem.tsx [hunk 1]
```diff
@@ -1,95 +1,94 @@
...
+};
 
-const PreferenceItem: React.FC<PrefProps> = ({
-  notificationTypeDetails,
-  value,
-  onUpdate
-}) => {
-  const { t } = useTranslation()
+const PreferenceItem: React.FC<PrefProps> = ({ category, notificationTypeDetails, value, onUpdate }) => {
+    const [openModal, setOpenModal] = useState<boolean>(false);
```
Comment (timidak) [inline src/components/notifications/preferences/PreferenceItem.tsx:16]: isModalOpen seems to be a better name, due to the fact that it is a boolean value. Same for the setter 😄

File: src/components/notifications/preferences/PreferenceItem.tsx [hunk 2]
```diff
@@ -1,95 +1,94 @@
...
-  )
-}
+    const handleToggle = (preferenceValue: boolean) => {
+        onUpdate(preferenceValue);
+    };
 
-export default PreferenceItem
+    return (
+        <>
+            <Box borderBottomWidth={1} borderBottomColor={'#F7F7F7'} py={3} width={width}>
```
Comment (timidak) [inline src/components/notifications/preferences/PreferenceItem.tsx:45]: The borderButtomColor seems to be a "magic number" here. Can we get it from the theme maybe? Or from a centralized place?

File: src/components/notifications/preferences/PreferenceItem.tsx [hunk 3]
```diff
@@ -1,95 +1,94 @@
...
+                <HStack alignItems="center" space={1}>
+                    {notificationTypeDetails?.icon && (
+                        <VStack>
+                            <Icon />
+                        </VStack>
+                    )}
+                    <VStack maxW={maxW}>
+                        <Text fontSize="md" mr="3" ellipsizeMode="tail" numberOfLines={2}>
+                            {t(notificationTypeDetails.title)}
+                            <>
```
Comment (timidak) [inline src/components/notifications/preferences/PreferenceItem.tsx:55]: You don't need the Fragment here, I guess.

**Review by timidak — COMMENTED** [review]
Just a few minor refinements

File: src/components/notifications/preferences/PreferenceItem.tsx (hunk 1 above)
Comment (timidak) [reply src/components/notifications/preferences/PreferenceItem.tsx:16]: setIsModalOpen for the setter

File: src/components/notifications/preferences/PreferenceItem.tsx [hunk 4]
```diff
@@ -1,95 +1,89 @@
...
+                <VStack maxW={maxW}>
+                    <Text fontSize="md" mr="3" ellipsizeMode="tail" numberOfLines={2}>
+                        {t(notificationTypeDetails.title)}
+                        {isMobile ? (
+                            <Box>
+                                <Pressable ml={1} onPress={() => setIsModalOpen(true)}>
+                                    <InformationBadge />
+                                </Pressable>
+                                <Modal bg="modalbg" isOpen={isModalOpen} onClose={() => setIsModalOpen(false)}>
+                                    <InformationModal onPressClose={() => setIsModalOpen(false)} category={category} />
```
Comment (jeffersondarcy) [inline src/components/notifications/preferences/PreferenceItem.tsx:61]: do you need `setIsModalOpen` twice?

File: src/components/notifications/preferences/PreferenceItem.tsx [hunk 5]
```diff
@@ -1,95 +1,89 @@
...
+                                <Circle rounded="full" bg="amber.700" size={4} ml={1}>
+                                    <Text color={'white'}>i</Text>
+                                </Circle>
+                            </Tooltip>
+                        )}
+                    </Text>
+                </VStack>
+                <Spacer />
+                <VStack>
+                    <Switch value={value} onToggle={() => handleToggle(!value)} />
```
Comment (jeffersondarcy) [inline src/components/notifications/preferences/PreferenceItem.tsx:82]: this has not change, but we might want improve it `onToggle={handleToggle}` looks better, can we avoid negation?

**Review by jeffersondarcy — APPROVED** [review]
Looks good! I have left two comments, but they are uncritical. Approved

File: src/components/notifications/preferences/PreferenceItem.tsx (hunk 4 above)
Comment (LomyW) [reply src/components/notifications/preferences/PreferenceItem.tsx:61]: good point :D

**Pull request merged**